utils/_context/containers.py

```diff
@@ -195,14 +199,23 @@ def execute_command(self, test, retries=10, interval=1_000_000_000, start_period
                     return
 
             except APIError as e:
-                logger.exception(f"Try #{i} failed")
-                pytest.exit(f"Command {cmd} failed for {self._container.name}: {e.explanation}", 1)
+                logger.exception(f"Try #{i} failed for {self._container.name}: {e.explanation}")
+                break
 
             except Exception as e:
                 logger.debug(f"Try #{i}: {e}")
 
             time.sleep(interval)
 
+        logs = "<cannot retrieve logs>"
+        try:
+            logs = self._container.logs(stdout=True, stderr=True)
+            logs = logs.decode("utf-8")
+        except:
+            pass
+
+        print(logs)
```

Comment:
why this? anyway we should use the logger

Reply:
When the healthcheck fails, this helps determine the cause of the failure.